src/main/resources/templates/startPage.html

```diff
@@ -12,23 +12,28 @@
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
 </head>
 <h1>Simple Talk</h1>
-<body>
-    <br />
+<body style="padding: 20px 40px">
     <form action="#"  method="GET">
         <label for="searchTerm">Search</label><input type="text" name="searchTerm" id="searchTerm"/>
-        <button name="submit" tyoe="submit" value="0">Search</button>
+        <button class="btn" name="submit" type="submit" value="0">Search</button>
     </form>
-    <br />
+    <br>
     <p>Simply say what you want to say!</p>
     <p th:text="${post}" />
 
     <form action="#" th:action="@{/savePost}" th:object="${post}" method="GET">
-        <label>PostID</label><input type="text" name="postID" th:field="*{postID}"/>
-        <label>UserID</label><input type="text" name="userID" th:field="*{userID}"/>
-        <label>UserName</label><input type="text" name="userName" th:field="*{userName}"/>
-        <label>Message</label><input type="text" name="message" th:field="*{message}"/>
-        <label>Likes</label><input type="text" name="likes" th:field="*{likes}"/>
-        <button name="button" type="submit" value="0">Post</button>
+        <label>PostID</label><br>
+        <input type="text" name="postID" th:field="*{postID}"/><br>
+        <label>UserID</label><br>
+        <input type="text" name="userID" th:field="*{userID}"/><br>
+        <label>UserName</label><br>
+        <input type="text" name="userName" th:field="*{userName}"/><br>
+        <label>Message</label><br>
+        <textarea name="message" rows="5" cols="50" th:field="*{message}">
+        </textarea><br>
+        <label>Likes</label><br>
+        <input type="text" name="likes" th:field="*{likes}"/><br><br>
+        <button class="btn btn-primary" name="button" type="submit" value="0">Post</button>
```

Comment:
Putting labels on a new line is a UI change, which is really up to the group...
None the less, a suggestion: use the placeholder attribute of the input tag, and remove the labels all together.  A cleaner, simpler interface.